apps/modernization-ui/src/apps/patient/add/extended/inputs/address/AddressMultiEntry.tsx

```diff
@@ -1,9 +1,9 @@
 import { AddressEntry } from 'apps/patient/data/entry';
 import { internalizeDate } from 'date';
-import { MultiValueEntry } from 'design-system/entry/multi-value/MultiValueEntry';
 import { Column } from 'design-system/table';
 import { AddressView } from './AddressView';
 import { AddressEntryFields } from 'apps/patient/data/address/AddressEntryFields';
+import { ValueView } from 'design-system/entry/multi-value/ValueView';
```

Comment:
Issue: I believe the name for the `MultiValueEntry` component was going to be `RepeatingBlock`, and `DataDisplay` was going to be updated to `ValueView`